resources/public/api/conf/5.0/schemas/foreign_property_cumulative_summary_retrieve/def1/response.json

```diff
@@ -0,0 +1,198 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "For TY 2025-26 and after",
+  "description": "This endpoint enables you to retrieve the cumulative period income and expenses for a foreign property business. A National Insurance number, business ID, and tax year are required. Please note that this endpoint is only available for tax years starting from 2025-26.",
+  "type": "object",
+  "properties": {
+    "submittedOn": {
+      "type": "string",
+      "description": "The date the foreign property were submitted, in the format YYYY-MM-DDThh:mm:ss.SSSZ",
+      "example": "2025-07-07T10:59:47.544Z"
+    },
```

Comment:
Should we include `"format": "date-time"` here, similar to what is specified in our specification `String (DateTime)` and the downstream specification?